### ISSUE TITLE
Does it work with NetCDF files using xr.apply_ufunc() ? 

### ISSUE THREAD

**Issue by SouhailAB** [e-baumer/standard_precip#14]

**Comment by e-baumer** [comment]
Yes. As of v2.0.0 this works cleanly with `xr.apply_ufunc` by wrapping the calculation per grid cell. A verified recipe, including the dask parallelized variant for large grids, is now in the [Gridded data section of the README](https://github.com/e-baumer/standard_precip#gridded-data-netcdf--xarray).

Native xarray support, where a DataArray would be accepted directly, is not planned for 2.0, but the recipe above is functionally equivalent. Closing this now. Feel free to reopen if the recipe does not cover your case.
